GDJS/GDJS/Extensions/Builtin/AsyncExtension.cpp

```diff
@@ -49,6 +49,11 @@ AsyncExtension::AsyncExtension() {
                 ? "const asyncObjectsList = "
                   "gdjs.LongLivedObjectsList.from(parentAsyncObjectsList);\n"
                 : "const asyncObjectsList = new gdjs.LongLivedObjectsList();\n";
+
+        asyncObjectsListBuilder +=
+            "asyncObjectsList.setLocalVariablesContainers(runtimeScene._"
+            "localVariables);\n";
```

Comment:
Nit: Don't add this to `asyncObjectsListBuilder`, this variable as the name implies is the code that will "build the async object lists". Instead, name this `backupLocalVariablesContainersCode` and add it at the end after/before asyncObjectsListBuilder